Interoperability/python/qsharp/__init__.py

```diff
@@ -16,6 +16,15 @@
 from abc import ABCMeta, abstractproperty
 from random import randrange
 
+# CLR Imports #
+import clr
```

Comment:
Moving this import means that the libraries of Quantum are now also part of the except.
While it should not matter that much, its a change in behaviour.